src/QtHardMon.cpp

```diff
@@ -1,26 +1,25 @@
 #include "QtHardMon.h"
+
 #include "ConfigFileReaderWriter.h"
+#include "Exceptions.h"
```

Comment:
QtHardMon is in a bad state as code formatting is concerned. We should have made a separate commit first, with just the auto-formatting changes, and no functional changes. Then it is not mixed with the functional changes which need review.

It would be too much work with too little benefit to separate that now. We (that includes me because I should have known) have to remember to do this in future tasks, and establish an according work flow.

Reply:
(Note: just resolve this comment after reading and continue with the merge)